packages/tests/tests/peer_exchange.node.spec.ts

```diff
@@ -72,9 +72,9 @@ describe("Peer Exchange", () => {
     });
 
     afterEach(async function () {
-      !!nwaku1 && nwaku1.stop();
-      !!nwaku2 && nwaku2.stop();
-      !!waku && waku.stop().catch((e) => console.log("Waku failed to stop", e));
+      !!nwaku1 && (await nwaku1.stop());
```

Comment:
If `nwaku1` fails to stop then it's possible to keep `nwaku2` live. 
Better to re-write it to `.stop().catch(noop)`